### PR TITLE
[Kafka] [ConsumerGroup] Increases additional dimension field limit

### PR DESCRIPTION
**Pull request by agithomas**
- Enhancement

## What does this PR do?

Increases the dimension field limit to 24 to support all dimensions

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #6278

### CODE REVIEW

File: packages/kafka/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.5.7"
+  changes:
+    - description: Added additional dimension fields.
+      type: enhancement
```
Comment (ritalwar) [inline packages/kafka/changelog.yml:5]: shouldn't this be bugfix?

**Review by ritalwar — APPROVED** [review]
LGTM!

**Comment by agithomas** [comment]
@lalit-satapathy , are we good to update kibana version to 8.6 for kafka, also considering upcoming beta release changes?

**Comment by Phoelsch** [comment]
> Hi! We just realized that we haven't looked into this PR in a while. We're sorry! We're labeling this issue as `Stale` to make it hit our filters and make sure we get back to it as soon as possible. In the meantime, it'd be extremely helpful if you could take a look at it as well and confirm its relevance. A simple comment with a nice emoji will be enough `:+1`. Thank you for your contribution!

 :+1:

**Comment by agithomas** [comment]
Closing this PR as the change is applied using a different PR. [Reference](https://github.com/elastic/integrations/blob/main/packages/kafka/data_stream/consumergroup/manifest.yml)

**Pull request closed without merging**